a3p-integration/proposals/b:restart-vats/README.md

```diff
@@ -0,0 +1,9 @@
+A demonstration of how to declare a CoreEvalProposal
+
+The console output will show errors, because the `restart-vats` proposal is not well maintained.
+
+The `submission` dir was built using:
+`packages/builders/scripts/vats/restart-vats.js`
+
+UNTIL https://github.com/Agoric/agoric-3-proposals/issues/87
```

Comment:
I am extremely uncomfortable with this approach in the context of an `a3p-integration` test. People will not know and forget to regenerate these bundles, which will get out of sync with the code in the repo.

At the very least, I would want CI to verify that the bundles are up to date, but then that begs the question of why CI can't generate the bundles automatically in the first place.

Is there really not a way to support a "spawn this command to generate bundles" mode? What in the local generation requires manual intervention that it can't be scripted?

Reply:
Sure it's possible. I understood that someone was blocked on the changes already here so I figured that can land after these other changes. But they don't actually depend on these changes so I'll drop this back into draft and make https://github.com/Agoric/agoric-3-proposals/issues/87 be part of its requirements.

There's other work here that's safe to land which I've pulled out into https://github.com/Agoric/agoric-sdk/pull/8839